packages/babel-parser/test/fixtures/typescript/scope/module-declaration-var/input.js

```diff
@@ -0,0 +1,5 @@
+declare class foo {
+}
+declare module 'bar' {
+  export var foo: any; // Identifier 'foo' has already been declared
```

Comment:
Does it work without `declare`?

Reply:
is that a valid syntax?

Reply:
Only with an identifier name instead of a string:
https://www.typescriptlang.org/play/#src=module%20foo%20%7B%0A%20%20%20%20var%20a%20%3D%202%3B%0A%7D%0A

Reply:
oh okay, added a test case for that